packages/node_modules/@webex/internal-plugin-conversation/test/integration/spec/verbs.js

```diff
@@ -208,6 +208,43 @@ describe('plugin-conversation', function () {
         .then((reason) => assert.equal(reason.status, 403)));
     });
 
+    describe('#post() with error handling', () => {
```

Comment:
Apologies, but it looks like you used the pattern set by the ['#leave() with id only'](https://github.com/webex/webex-js-sdk/blob/c378d07f8622078603384d17e90ef3d04e19a2ff/packages/node_modules/%40webex/internal-plugin-conversation/test/integration/spec/verbs.js#L190) block which is incorrect and must have slipped through previously.
 Could you move this describe block within the `#post()` describe block currently on line 248? Suggested structure:

```js
describe('#post()', () => {
...
  describe('when encryption key needs to rotate', () => {
    it('rotates the key and posts a plain text', () => {
```

Reply:
Okay, will do